frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.tsx

```diff
@@ -86,17 +90,37 @@ export const ClassificationForm: React.FC<IClassificationFormProps> = ({
   encumbranceField,
 }) => {
   const formikProps = useFormikContext();
+  const keycloak = useKeycloakWrapper();
   let surplusActiveOrEncumbered =
     getIn(formikProps.values, field) === Classifications.SurplusEncumbered ||
     getIn(formikProps.values, field) === Classifications.SurplusActive;
 
   /** classId based on current formik values to determine which classsification information box to display */
   let classId = getIn(formikProps.values, field);
 
+  /** determine how to filter the classifications */
+  const determineFilter = (val: Number) => {
+    /** non SRES users cannot see subdivided */
+    if (!keycloak.hasClaim(Claims.ADMIN_PROPERTIES)) {
```

Comment:
just a small style comment, if possible if true else false is generally easier for people to read. Not a hard and fast rule of course.

Reply:
Ah makes sense, will adjust to keep with best practices!